CONTRIBUTING.MD

```diff
@@ -34,6 +34,42 @@ Example:
 }
 ```
 
+### Creating Components on the `fastore/ui`
+
+You can create "boilerplates files" for your new component using the following command:
+
+- run `yarn generate`
+
+```bash
+# What is your component name?
+
+ComponentName
+
+#  What is your component atomic group?  (Use arrow keys)
+
+> atoms
+> molecules
+> organisms
+```
+
+- It will automatically generate the files below:
+
+```
+✔  ++ faststore/packages/ui/src/atoms/ComponentName/ComponentName.tsx
+✔  ++ faststore/packages/ui/src/atoms/ComponentName/index.tsx
+✔  ++ faststore/themes/theme-b2c-tailwind/src/atoms/component-name.css
+✔  ++ faststore/packages/ui/src/atoms/ComponentName/ComponentName.test.tsx
+✔  ++ faststore/packages/ui/src/atoms/ComponentName/stories/ComponentName.stories.tsx
+✔  ++ faststore/packages/ui/src/atoms/ComponentName/stories/ComponentName.mdx
+✔  _+ faststore/packages/ui/src/index.ts
+```
+
+And that is it! You will be ready to work on your component. 🎉
+
+> For the component name, please follow CamelCase pattern, and do not use `-`, i.e., Component-Name.
+
+> Please try to not edit // Atoms, // Molecules, // Organisms comments on `packages/ui/scr/index.ts` [file](https://github.com/vtex/faststore/blob/b1badead35dc43b41f25480a66f4e81cc9a8e22a/packages/ui/src/index.ts), because we are using it as [pattern](https://github.com/vtex/faststore/blob/443cbd117e34ea1702a2994ca4ad0b2733f17189/generators/plopfile.ts#L62) to update it. For more details, check [here](https://github.com/vtex/faststore/pull/1039).
```

Comment:
```suggestion
> Please do not edit `// Atoms`, `// Molecules` and `// Organisms` comments on `packages/ui/scr/index.ts` [file](https://github.com/vtex/faststore/blob/b1badead35dc43b41f25480a66f4e81cc9a8e22a/packages/ui/src/index.ts), because we are using it as a [pattern](https://github.com/vtex/faststore/blob/443cbd117e34ea1702a2994ca4ad0b2733f17189/generators/plopfile.ts#L62) to update the file. For more details, check [here](https://github.com/vtex/faststore/pull/1039).
```